lib/pf/services/manager/etcd.pm

```diff
@@ -0,0 +1,101 @@
+package pf::services::manager::etcd;
+=head1 NAME
+
+pf::services::manager::etcd
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::services::manager::etcd
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+use pf::cluster;
+use pf::config qw(
+    %Config
+    $management_network
+);
+use pf::cluster qw(
+    @cluster_servers
+);
+
+use pf::file_paths qw(
+    $generated_conf_dir
+    $install_dir
+    $conf_dir
+    $var_dir
+);
+use pf::log;
+
+use pf::util;
+
+extends 'pf::services::manager';
+
+has '+name' => ( default => sub { 'etcd' } );
+
+sub isManaged {
+    my ($self) = @_;
+    return  isenabled($Config{'services'}{'go_dhcpd'}) && $self->SUPER::isManaged();
```

Comment:
go_dhcpd  doesn't exist anymore

And anyway IMO, if etcd is explicitely disabled in pf.conf, I'd leave it disabled so this subroutine could be removed